### ISSUE TITLE
NotImplementedException inside IsEmptyOrWhiteSpace

### ISSUE THREAD

**Issue by jjonescz** [dotnet/aspnetcore#20060]
### Bug description
Property implementation `DefaultTagHelperContent.IsWhiteSpaceOrEmpty` throws `NotImplementedException`.

### How to reproduce
See [mimimum viable example](https://github.com/jjonescz/WhiteSpaceWriter-bug). The exception is thrown on [line 24 in `Pages/Index.cshtml`](https://github.com/jjonescz/WhiteSpaceWriter-bug/blob/master/Pages/Index.cshtml#L23-L25).

### Further technical details
- ASP.NET Core version: 3.1.200
- Output of `dotnet --info`:
  ```txt
  .NET Core SDK (reflecting any global.json):
  Version:   3.1.200
  Commit:    c5123d973b

  Runtime Environment:
  OS Name:     Windows
  OS Version:  10.0.18363
  OS Platform: Windows
  RID:         win10-x64
  Base Path:   C:\Program Files\dotnet\sdk\3.1.200\

  Host (useful for support):
    Version: 3.1.2
    Commit:  916b5cba26

  .NET Core SDKs installed:
    3.0.100-preview7-012821 [C:\Program Files\dotnet\sdk]
    3.0.100-preview8-013656 [C:\Program Files\dotnet\sdk]
    3.0.100 [C:\Program Files\dotnet\sdk]
    3.1.100 [C:\Program Files\dotnet\sdk]
    3.1.200 [C:\Program Files\dotnet\sdk]

  .NET Core runtimes installed:
    Microsoft.AspNetCore.All 2.1.4 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.All]
    Microsoft.AspNetCore.All 2.1.16 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.All]
    Microsoft.AspNetCore.App 2.1.4 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
    Microsoft.AspNetCore.App 2.1.16 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
    Microsoft.AspNetCore.App 3.0.0-preview7.19365.7 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
    Microsoft.AspNetCore.App 3.0.0-preview8.19405.7 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
    Microsoft.AspNetCore.App 3.0.0 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
    Microsoft.AspNetCore.App 3.0.3 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
    Microsoft.AspNetCore.App 3.1.0 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
    Microsoft.AspNetCore.App 3.1.2 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
    Microsoft.NETCore.App 2.1.16 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
    Microsoft.NETCore.App 3.0.0-preview7-27912-14 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
    Microsoft.NETCore.App 3.0.0-preview8-28405-07 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
    Microsoft.NETCore.App 3.0.0 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
    Microsoft.NETCore.App 3.0.3 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
    Microsoft.NETCore.App 3.1.0 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
    Microsoft.NETCore.App 3.1.2 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
    Microsoft.WindowsDesktop.App 3.0.0-preview7-27912-14 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
    Microsoft.WindowsDesktop.App 3.0.0-preview8-28405-07 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
    Microsoft.WindowsDesktop.App 3.0.0 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
    Microsoft.WindowsDesktop.App 3.0.3 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
    Microsoft.WindowsDesktop.App 3.1.0 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
    Microsoft.WindowsDesktop.App 3.1.2 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]

  To install additional .NET Core runtimes or SDKs:
    https://aka.ms/dotnet-download
  ```
- IDE: Visual Studio 2019, version 16.5.0

**Comment by mkArtakMSFT** [comment]
Thanks for contacting us.
@NTaylorMullen can you please look into this? Thanks!

**Comment by mkArtakMSFT** [comment]
We've moved this issue to the Backlog milestone. This means that it is not going to be worked on for the coming release. We will reassess the backlog following the current release and consider this item at that time. To learn more about our issue management process and to have better expectation regarding different types of issues you can read our [Triage Process](https://github.com/dotnet/aspnetcore/blob/master/docs/TriageProcess.md).

**Comment by fagnercarvalho** [comment]
@mkArtakMSFT, I believe this issue can be closed since #20061 was merged.

**Comment by pranavkm** [comment]
Resolved as part of https://github.com/dotnet/aspnetcore/pull/20061.
